objectbox/lib/flatbuffers/flat_buffers.dart

```diff
@@ -745,14 +774,15 @@ class Builder {
   /// Additionally allocate the specified `additionalBytes`. Update the current
   /// tail pointer to point at the allocated space.
   @pragma('vm:prefer-inline')
-  void _prepare(int size, int count, {int additionalBytes = 0}) {
+  void _prepare(int size, int count,
+      {int additionalBytes = 0, bool align = true}) {
```

Comment:
This appears to be never used with `align = false`? Can this change be removed?

Reply:
ah, yeah, leftover from an inbetween solution I tried (converting from the end of the ascii string and switching to utf converter for the rest. Couldn't get it to work because of alignment...

Reply:
removed